sdk/go/pulumiservice/environment.go

```diff
@@ -0,0 +1,250 @@
+// Code generated by pulumi-language-go DO NOT EDIT.
```

Comment:
This diff is surprising in this PR. Did we forget to include this earlier?

Reply:
Yep, the go files only didn't generate for Environment resources, Stoo pinged me about it. Looking into it, a while ago I updated .gitignore and it's covering too much, which caused this. I'll have to address the gitignore issue, but figured should include the missed files asap

Reply:
Can we pull this out into a separate PR and get that merged and released? I'd rather not hold that up on this PR.